app/controllers/wikies_controller.rb

```diff
@@ -8,7 +8,7 @@ def search_results
   end
 
   def index
-    @allowed_collaborations = current_user.collaborations.map(&:wiky)
+    @wikies_allowed_to_collaborate_on = current_user.collaborations.map(&:wiky)
```

Comment:
Good job on renaming the variable. 

The line can be simplified further to not use `map`.

Knowing we've got line 8 in the `User` model declaring the `collaborated_wikys` association: https://github.com/LuisMenjivar/wiki/blob/60d8b9c8b42f020ac30fd280c31011461aaad31b/app/models/user.rb#L8

`current_user` is an instance of `User`, so the `collaborated_wikys` association is available here:

```ruby
@wikies_allowed_to_collaborate_on = current_user.collaborated_wikys
```